### PR TITLE
fix(TBD-12577) removing old HDPs in master

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by sponomarova — CHANGES_REQUESTED** [review]
1. I though hdp3x will be removed on master. Right?
2. Please fix unit tests. 
3. Please do clean-up in studio-full-master repository.

**Comment by sponomarova** [comment]
> 1. I though hdp3x will be removed on master. Right?
> 2. Please fix unit tests.
> 3. Please do clean-up in studio-full-master repository.

Realized that remark #1 is not valid but #2 and #3 remain. Need to be fixed.

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp3xxTemplate.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "Hortonworks HDP3.X.X template",
   "distribution": "HORTONWORKS",
   "templateId": "HDP3xxDistributionTemplate",
-  "baseVersion": "3.0.0",
+  "baseVersion": "3.1.0",
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp3xxTemplate.json:6]: Why do we need this change ?

File: main/plugins/org.talend.hadoop.distribution.hdpx/src/org/talend/hadoop/distribution/hdpx/DynamicHDPDistribution.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ protected IDynamicDistributionTemplate initTemplate(DynamicPluginAdapter pluginA
     public List<String> getSupportedTemplateIds(IDynamicMonitor monitor) throws Exception {
         List<String> templateIds = new ArrayList<>();
 
-        templateIds.add(HDP2xxDistributionTemplate.TEMPLATE_ID);
+        // templateIds.add(HDP2xxDistributionTemplate.TEMPLATE_ID);
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.hdpx/src/org/talend/hadoop/distribution/hdpx/DynamicHDPDistribution.java:54]: To be removed

File: test/plugins/org.talend.hadoop.distribution.test/src/org/talend/hadoop/distribution/test/helper/HadoopDistributionsHelperTest.java [hunk 3]
```diff
@@ -30,14 +30,14 @@
     public void testGetHadoopDistributionsDisplay_Sorted() {
         String[] hadoopDistributionsDisplay = HadoopDistributionsHelper.HADOOP.getDistributionsDisplay(false);
         Assert.assertArrayEquals(new String[] { "Amazon EMR", "Cloudera", "Databricks", "Google Cloud Dataproc",
-                "HortonWorks", "MapR", "Microsoft HD Insight", "Qubole"}, hadoopDistributionsDisplay);
+                "MapR", "Microsoft HD Insight", "Qubole"}, hadoopDistributionsDisplay);
```
Comment (lbourgeois) [inline test/plugins/org.talend.hadoop.distribution.test/src/org/talend/hadoop/distribution/test/helper/HadoopDistributionsHelperTest.java:33]: No clear why we don't have HortonWorks anymore as we still have HDP 3 in Studio

File: test/plugins/org.talend.repository.hadoopcluster.test/src/main/java/org/talend/repository/hadoopcluster/conf/RetrieveLocalConfsServiceTest.java [hunk 4]
```diff
@@ -20,27 +31,16 @@
...
-import java.util.List;
-import java.util.Map;
-
-import static org.junit.Assert.*;
-
 @Ignore("failed on tycho with NullPointerException")
 public class RetrieveLocalConfsServiceTest {
 
-    static IRetrieveConfsService confsService;
+//    static IRetrieveConfsService confsService;
```
Comment (lbourgeois) [inline test/plugins/org.talend.repository.hadoopcluster.test/src/main/java/org/talend/repository/hadoopcluster/conf/RetrieveLocalConfsServiceTest.java:43]: To be cleaned + same on other commented lined

**Comment by sponomarova** [comment]
Needed changed will be done in scope of:
https://github.com/Talend/tbd-studio-se/pull/2073

**Pull request closed without merging**